File: fortran/problem_1.f90
program  problem_1
    integer i
    integer total
    total = 0
    do i=0,999
        if (MODULO(i, 3) == 0 .OR. MODULO(i, 5) == 0) then
            total = total + i
        end if
    end  do
    print *,"Problem 1:", total
end program problem_1

File: c/problem_1.c
#include <stdio.h>

int main() 
{
    int total;
    for(int x = 0; x < 1000; x++) {
        if (x % 3 == 0 || x % 5 == 0) {
            total += x;
        }
    }
    printf("Problem 1: %i\n", total);
    return 0;
}